src/services/payment.services.ts

```diff
@@ -95,6 +96,10 @@ export const orderItems = async (cart: cartModelAttributes) => {
 		};
 		await insert_function<salesModelAttributes>("Sales", "create", sale_data);
 	}
+	await database_models.Cart.update(
+		{ products: [] },
+		{ where: { id: cart.id } },
```

Comment:
@cyusasnave , you are updating product only, But you forget to make total cart to 0.